Firmware/firmwareLiBCM/src/key.cpp

```diff
@@ -99,3 +103,28 @@ uint8_t key_getSampledState(void)
 }
 
 /////////////////////////////////////////////////////////////////////////////////////////
+
+void keyOn_coldBootTasks(void)
```

Comment:
Suggestion:
This sequence might be better placed in `firmwareLiBCM.ino` itself--similar to how `setup()` and `loop()` define their respective sequences.

From an "object" perspective, a `key` says whether it's `on`, `off`, and if it has changed; but it shouldn't perform the system consequences of those states.

This also applies to `key_handleKeyEvent_off()`, `key_handleKeyEvent_on()`, and `void key_stateChangeHandler()`.